store/reducers/aliasData.ts

```diff
@@ -18,7 +18,7 @@ export const aliasData = (state: State = {}, action: AnyAction) => {
 
   switch (action.type) {
     case HYDRATE:
-      return { ...action.payload.aliasData, ...state };
+      return { ...state, ...action.payload.aliasData };
```

Comment:
Is this the final issue you had to resolve? flipping the loading state?

Reply:
Part of it. This was actually reverting something I was trying. Main issue was the way the api endpoints where returning (actually not returning) responses.